README.md

```diff
@@ -8,10 +8,12 @@
 
 The official wallet of the fastest modular execution layer: [Fuel](https://fuel.network).
 
-FuelWallet is a crypto wallet used to interact with the Fuel Network. This project enables users to access their assets and interact with decentralized applications through a browser extension.
+Fuel Wallet is a crypto wallet used to interact with the Fuel Network. This project enables users to access their assets and interact with decentralized applications through a browser extension. 
 
 [![Install Wallet](packages/docs/public/install-button.png)](https://wallet.fuel.network/docs/install/)
 
+For latest updates donwload the [Fuel Wallet Development](https://next-wallet.fuel.network/docs/install/) this version is made for developers. It can be unstable and have issues or bugs.
```

Comment:
```suggestion
For latest updates, download the [Fuel Wallet Development](https://next-wallet.fuel.network/docs/install/). This version is made for developers, so note that it can be unstable or buggy.
```